Neos.Flow/Classes/Persistence/Doctrine/Query.php

```diff
@@ -69,7 +68,7 @@ class Query implements QueryInterface
     /**
      * @var array
      */
-    protected $orderings;
+    protected $orderings = [];
 
     /**
      * @var integer
```

Comment:
```suggestion
     * @var integer|null
```
Same for the offset below, but GH doesn't allow suggestions/comments on lines outside the diff lines